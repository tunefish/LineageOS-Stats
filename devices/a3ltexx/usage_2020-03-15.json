{
  "14.1-20170204-unofficial-a3ltexx": 37,
  "14.1-20170330-unofficial-a3ltexx": 23,
  "14.1-20170525-unofficial-a3ltexx": 1316
}