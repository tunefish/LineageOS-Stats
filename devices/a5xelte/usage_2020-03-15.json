{
  "15.1-20190426-unofficial-a5xelte": 7,
  "15.1-20190429-unofficial-a5xelte": 1,
  "15.1-20190504-unofficial-a5xelte": 2,
  "15.1-20190517-unofficial-a5xelte": 3,
  "15.1-20190526-unofficial-a5xelte": 6,
  "15.1-20190528-unofficial-a5xelte": 1,
  "15.1-20190623-unofficial-a5xelte": 5,
  "15.1-20190709-unofficial-a5xelte": 52,
  "16.0-20190121-unofficial-a5xelte": 32,
  "16.0-20190625-unofficial-a5xelte": 9,
  "16.0-20190627-unofficial-a5xelte": 3,
  "16.0-20190707-unofficial-a5xelte": 4,
  "16.0-20190723-unofficial-a5xelte": 14,
  "16.0-20190811-unofficial-a5xelte": 4,
  "16.0-20190816-unofficial-a5xelte": 2,
  "16.0-20190818-unofficial-a5xelte": 47,
  "16.0-20190822-nightly-a5xelte": 1,
  "16.0-20190823-nightly-a5xelte": 1,
  "16.0-20190824-nightly-a5xelte": 1,
  "16.0-20190825-nightly-a5xelte": 1,
  "16.0-20190827-nightly-a5xelte": 1,
  "16.0-20190831-nightly-a5xelte": 2,
  "16.0-20190902-nightly-a5xelte": 1,
  "16.0-20190903-nightly-a5xelte": 2,
  "16.0-20190904-nightly-a5xelte": 3,
  "16.0-20190905-nightly-a5xelte": 2,
  "16.0-20190907-nightly-a5xelte": 2,
  "16.0-20190908-nightly-a5xelte": 2,
  "16.0-20190909-unofficial-a5xelte": 2,
  "16.0-20190910-nightly-a5xelte": 5,
  "16.0-20190913-nightly-a5xelte": 4,
  "16.0-20190914-nightly-a5xelte": 2,
  "16.0-20190915-nightly-a5xelte": 2,
  "16.0-20190917-nightly-a5xelte": 3,
  "16.0-20190918-nightly-a5xelte": 1,
  "16.0-20190921-nightly-a5xelte": 2,
  "16.0-20190923-nightly-a5xelte": 1,
  "16.0-20190925-nightly-a5xelte": 7,
  "16.0-20190926-nightly-a5xelte": 1,
  "16.0-20190928-nightly-a5xelte": 2,
  "16.0-20190930-nightly-a5xelte": 3,
  "16.0-20191002-nightly-a5xelte": 3,
  "16.0-20191004-nightly-a5xelte": 2,
  "16.0-20191005-nightly-a5xelte": 1,
  "16.0-20191006-nightly-a5xelte": 2,
  "16.0-20191007-microg-a5xelte": 1,
  "16.0-20191007-nightly-a5xelte": 4,
  "16.0-20191008-nightly-a5xelte": 2,
  "16.0-20191010-nightly-a5xelte": 1,
  "16.0-20191011-nightly-a5xelte": 3,
  "16.0-20191012-nightly-a5xelte": 2,
  "16.0-20191013-nightly-a5xelte": 4,
  "16.0-20191015-nightly-a5xelte": 3,
  "16.0-20191016-nightly-a5xelte": 1,
  "16.0-20191017-nightly-a5xelte": 4,
  "16.0-20191018-nightly-a5xelte": 1,
  "16.0-20191019-nightly-a5xelte": 2,
  "16.0-20191020-nightly-a5xelte": 2,
  "16.0-20191022-nightly-a5xelte": 7,
  "16.0-20191023-nightly-a5xelte": 1,
  "16.0-20191024-nightly-a5xelte": 1,
  "16.0-20191026-nightly-a5xelte": 4,
  "16.0-20191027-nightly-a5xelte": 4,
  "16.0-20191029-nightly-a5xelte": 2,
  "16.0-20191030-nightly-a5xelte": 3,
  "16.0-20191101-nightly-a5xelte": 4,
  "16.0-20191102-nightly-a5xelte": 4,
  "16.0-20191103-nightly-a5xelte": 1,
  "16.0-20191104-nightly-a5xelte": 4,
  "16.0-20191105-nightly-a5xelte": 5,
  "16.0-20191106-nightly-a5xelte": 3,
  "16.0-20191107-nightly-a5xelte": 1,
  "16.0-20191108-nightly-a5xelte": 4,
  "16.0-20191109-nightly-a5xelte": 4,
  "16.0-20191110-nightly-a5xelte": 18,
  "16.0-20191111-microg-a5xelte": 1,
  "16.0-20191112-nightly-a5xelte": 8,
  "16.0-20191114-nightly-a5xelte": 11,
  "16.0-20191115-nightly-a5xelte": 3,
  "16.0-20191116-nightly-a5xelte": 10,
  "16.0-20191117-nightly-a5xelte": 13,
  "16.0-20191118-nightly-a5xelte": 14,
  "16.0-20191119-nightly-a5xelte": 6,
  "16.0-20191120-nightly-a5xelte": 1,
  "16.0-20191121-nightly-a5xelte": 11,
  "16.0-20191122-nightly-a5xelte": 20,
  "16.0-20191124-nightly-a5xelte": 28,
  "16.0-20191125-nightly-a5xelte": 18,
  "16.0-20191126-nightly-a5xelte": 53,
  "16.0-20191127-nightly-a5xelte": 18,
  "16.0-20191127-unofficial-a5xelte": 1,
  "16.0-20191128-nightly-a5xelte": 30,
  "16.0-20191129-nightly-a5xelte": 23,
  "16.0-20191130-nightly-a5xelte": 42,
  "16.0-20191201-nightly-a5xelte": 27,
  "16.0-20191202-microg-a5xelte": 2,
  "16.0-20191202-nightly-a5xelte": 46,
  "16.0-20191204-nightly-a5xelte": 26,
  "16.0-20191205-nightly-a5xelte": 33,
  "16.0-20191206-nightly-a5xelte": 22,
  "16.0-20191207-nightly-a5xelte": 16,
  "16.0-20191208-nightly-a5xelte": 35,
  "16.0-20191209-nightly-a5xelte": 31,
  "16.0-20191210-nightly-a5xelte": 34,
  "16.0-20191211-nightly-a5xelte": 37,
  "16.0-20191213-nightly-a5xelte": 34,
  "16.0-20191214-nightly-a5xelte": 41,
  "16.0-20191215-nightly-a5xelte": 7,
  "16.0-20191216-nightly-a5xelte": 21,
  "16.0-20191217-nightly-a5xelte": 25,
  "16.0-20191218-nightly-a5xelte": 31,
  "16.0-20191219-nightly-a5xelte": 27,
  "16.0-20191220-nightly-a5xelte": 12,
  "16.0-20191220-unofficial-a5xelte": 2,
  "16.0-20191221-nightly-a5xelte": 56,
  "16.0-20191222-nightly-a5xelte": 50,
  "16.0-20191223-microg-a5xelte": 1,
  "16.0-20191223-unofficial-a5xelte": 2,
  "16.0-20191224-nightly-a5xelte": 80,
  "16.0-20191224-unofficial-a5xelte": 2,
  "16.0-20191227-nightly-a5xelte": 25,
  "16.0-20191228-nightly-a5xelte": 18,
  "16.0-20191229-nightly-a5xelte": 19,
  "16.0-20191230-microg-a5xelte": 1,
  "16.0-20191230-nightly-a5xelte": 27,
  "16.0-20191230-unofficial-a5xelte": 3,
  "16.0-20191231-nightly-a5xelte": 25,
  "16.0-20200101-nightly-a5xelte": 19,
  "16.0-20200102-nightly-a5xelte": 18,
  "16.0-20200103-nightly-a5xelte": 47,
  "16.0-20200104-nightly-a5xelte": 7,
  "16.0-20200105-nightly-a5xelte": 17,
  "16.0-20200106-microg-a5xelte": 3,
  "16.0-20200106-nightly-a5xelte": 34,
  "16.0-20200107-nightly-a5xelte": 55,
  "16.0-20200109-nightly-a5xelte": 26,
  "16.0-20200110-nightly-a5xelte": 20,
  "16.0-20200111-nightly-a5xelte": 12,
  "16.0-20200112-nightly-a5xelte": 26,
  "16.0-20200113-microg-a5xelte": 2,
  "16.0-20200113-nightly-a5xelte": 16,
  "16.0-20200114-nightly-a5xelte": 15,
  "16.0-20200115-nightly-a5xelte": 26,
  "16.0-20200116-nightly-a5xelte": 34,
  "16.0-20200117-nightly-a5xelte": 22,
  "16.0-20200118-nightly-a5xelte": 38,
  "16.0-20200119-nightly-a5xelte": 41,
  "16.0-20200120-microg-a5xelte": 1,
  "16.0-20200120-nightly-a5xelte": 53,
  "16.0-20200121-nightly-a5xelte": 14,
  "16.0-20200122-nightly-a5xelte": 33,
  "16.0-20200123-nightly-a5xelte": 25,
  "16.0-20200124-nightly-a5xelte": 48,
  "16.0-20200126-nightly-a5xelte": 30,
  "16.0-20200127-microg-a5xelte": 8,
  "16.0-20200127-nightly-a5xelte": 14,
  "16.0-20200128-nightly-a5xelte": 15,
  "16.0-20200129-nightly-a5xelte": 24,
  "16.0-20200130-nightly-a5xelte": 29,
  "16.0-20200131-nightly-a5xelte": 9,
  "16.0-20200201-nightly-a5xelte": 28,
  "16.0-20200202-nightly-a5xelte": 17,
  "16.0-20200203-microg-a5xelte": 6,
  "16.0-20200203-nightly-a5xelte": 18,
  "16.0-20200204-nightly-a5xelte": 25,
  "16.0-20200205-nightly-a5xelte": 27,
  "16.0-20200206-nightly-a5xelte": 15,
  "16.0-20200207-nightly-a5xelte": 34,
  "16.0-20200208-nightly-a5xelte": 28,
  "16.0-20200209-nightly-a5xelte": 59,
  "16.0-20200210-microg-a5xelte": 3,
  "16.0-20200212-nightly-a5xelte": 13,
  "16.0-20200213-nightly-a5xelte": 24,
  "16.0-20200214-nightly-a5xelte": 23,
  "16.0-20200215-nightly-a5xelte": 23,
  "16.0-20200216-nightly-a5xelte": 16,
  "16.0-20200217-microg-a5xelte": 1,
  "16.0-20200217-nightly-a5xelte": 22,
  "16.0-20200218-nightly-a5xelte": 17,
  "16.0-20200219-nightly-a5xelte": 14,
  "16.0-20200220-nightly-a5xelte": 14,
  "16.0-20200221-nightly-a5xelte": 19,
  "16.0-20200222-nightly-a5xelte": 25,
  "16.0-20200223-nightly-a5xelte": 16,
  "16.0-20200224-nightly-a5xelte": 25,
  "17.0-20191119-unofficial-a5xelte": 1,
  "17.0-20191217-unofficial-a5xelte": 1,
  "17.0-20191222-unofficial-a5xelte": 2,
  "17.0-20191223-unofficial-a5xelte": 2,
  "17.0-20191224-unofficial-a5xelte": 4,
  "17.0-20191225-unofficial-a5xelte": 1,
  "17.0-20191230-unofficial-a5xelte": 2,
  "17.0-20191231-unofficial-a5xelte": 3,
  "17.0-20200102-unofficial-a5xelte": 5,
  "17.0-20200106-unofficial-a5xelte": 3,
  "17.0-20200108-unofficial-a5xelte": 6,
  "17.0-20200118-unofficial-a5xelte": 3,
  "17.0-20200215-unofficial-a5xelte": 9,
  "17.1-20200117-unofficial-a5xelte": 45,
  "17.1-20200119-unofficial-a5xelte": 1,
  "17.1-20200121-unofficial-a5xelte": 121,
  "17.1-20200125-unofficial-a5xelte": 1,
  "17.1-20200204-unofficial-a5xelte": 1,
  "17.1-20200207-unofficial-a5xelte": 2,
  "17.1-20200208-unofficial-a5xelte": 24,
  "17.1-20200210-unofficial-a5xelte": 34,
  "17.1-20200216-unofficial-a5xelte": 39,
  "17.1-20200219-unofficial-a5xelte": 2,
  "17.1-20200220-unofficial-a5xelte": 1,
  "17.1-20200221-unofficial-a5xelte": 6,
  "17.1-20200222-unofficial-a5xelte": 49,
  "17.1-20200225-unofficial-a5xelte": 3
}