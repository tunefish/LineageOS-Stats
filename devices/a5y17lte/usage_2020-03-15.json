{
  "14.1-20170722-unofficial-a5y17lte": 1,
  "14.1-20170811-unofficial-a5y17lte": 4,
  "14.1-20180103_133029-unofficial-a5y17lte": 2,
  "14.1-20180208-unofficial-a5y17lte": 1,
  "14.1-20180213-unofficial-a5y17lte": 7,
  "14.1-20180324-unofficial-a5y17lte": 1,
  "14.1-20180325-unofficial-a5y17lte": 19,
  "14.1-20180405-unofficial-a5y17lte": 3,
  "14.1-20180418-nightly-a5y17lte": 1,
  "14.1-20180425-nightly-a5y17lte": 1,
  "14.1-20180509-nightly-a5y17lte": 2,
  "14.1-20180516-nightly-a5y17lte": 1,
  "14.1-20180523-nightly-a5y17lte": 2,
  "14.1-20180530-nightly-a5y17lte": 3,
  "14.1-20180606-nightly-a5y17lte": 3,
  "14.1-20180613-nightly-a5y17lte": 60,
  "14.1-20180620-nightly-a5y17lte": 1,
  "14.1-20180627-nightly-a5y17lte": 4,
  "14.1-20180704-nightly-a5y17lte": 1,
  "14.1-20180711-nightly-a5y17lte": 1,
  "14.1-20180718-nightly-a5y17lte": 2,
  "14.1-20180725-nightly-a5y17lte": 3,
  "14.1-20180801-nightly-a5y17lte": 3,
  "14.1-20180808-nightly-a5y17lte": 6,
  "14.1-20180815-nightly-a5y17lte": 3,
  "14.1-20180822-nightly-a5y17lte": 1,
  "14.1-20180829-microg-a5y17lte": 1,
  "14.1-20180829-nightly-a5y17lte": 4,
  "14.1-20180912-nightly-a5y17lte": 2,
  "14.1-20180919-nightly-a5y17lte": 3,
  "14.1-20180926-nightly-a5y17lte": 89,
  "14.1-20181003-nightly-a5y17lte": 5,
  "14.1-20181017-nightly-a5y17lte": 1,
  "14.1-20181031-nightly-a5y17lte": 2,
  "14.1-20181107-nightly-a5y17lte": 2,
  "14.1-20181117-nightly-a5y17lte": 14,
  "14.1-20190117-nightly-a5y17lte": 13,
  "14.1-20190217-nightly-a5y17lte": 63,
  "14.1-20190302-microg-a5y17lte": 14,
  "14.1-20191118-unofficial-a5y17lte": 1,
  "14.1-20191124-unofficial-a5y17lte": 1,
  "15.1-20180223-unofficial-a5y17lte": 114,
  "15.1-20180804-unofficial-a5y17lte": 6,
  "15.1-20180902-unofficial-a5y17lte": 4,
  "15.1-20190516-unofficial-a5y17lte": 1,
  "15.1-20190524-unofficial-a5y17lte": 2,
  "15.1-20190702-unofficial-a5y17lte": 1,
  "15.1-20190823-unofficial-a5y17lte": 1,
  "16.0-20190610-unofficial-a5y17lte": 2,
  "16.0-20190707-unofficial-a5y17lte": 2,
  "16.0-20190801-nightly-a5y17lte": 1,
  "16.0-20190802-nightly-a5y17lte": 1,
  "16.0-20190804-nightly-a5y17lte": 2,
  "16.0-20190805-nightly-a5y17lte": 2,
  "16.0-20190806-nightly-a5y17lte": 1,
  "16.0-20190807-nightly-a5y17lte": 1,
  "16.0-20190808-nightly-a5y17lte": 1,
  "16.0-20190809-nightly-a5y17lte": 1,
  "16.0-20190813-nightly-a5y17lte": 1,
  "16.0-20190815-nightly-a5y17lte": 4,
  "16.0-20190816-nightly-a5y17lte": 3,
  "16.0-20190817-nightly-a5y17lte": 3,
  "16.0-20190818-nightly-a5y17lte": 2,
  "16.0-20190819-nightly-a5y17lte": 3,
  "16.0-20190822-nightly-a5y17lte": 1,
  "16.0-20190824-nightly-a5y17lte": 1,
  "16.0-20190826-nightly-a5y17lte": 1,
  "16.0-20190830-nightly-a5y17lte": 1,
  "16.0-20190831-nightly-a5y17lte": 16,
  "16.0-20190901-nightly-a5y17lte": 6,
  "16.0-20190903-nightly-a5y17lte": 4,
  "16.0-20190904-nightly-a5y17lte": 15,
  "16.0-20190905-nightly-a5y17lte": 1,
  "16.0-20190906-nightly-a5y17lte": 1,
  "16.0-20190907-nightly-a5y17lte": 1,
  "16.0-20190909-unofficial-a5y17lte": 1,
  "16.0-20190910-nightly-a5y17lte": 2,
  "16.0-20190912-nightly-a5y17lte": 1,
  "16.0-20190913-nightly-a5y17lte": 6,
  "16.0-20190915-nightly-a5y17lte": 2,
  "16.0-20190916-nightly-a5y17lte": 2,
  "16.0-20190917-nightly-a5y17lte": 1,
  "16.0-20190918-nightly-a5y17lte": 1,
  "16.0-20190919-nightly-a5y17lte": 1,
  "16.0-20190921-nightly-a5y17lte": 2,
  "16.0-20190922-nightly-a5y17lte": 2,
  "16.0-20190924-nightly-a5y17lte": 2,
  "16.0-20190925-nightly-a5y17lte": 4,
  "16.0-20190926-nightly-a5y17lte": 1,
  "16.0-20190927-nightly-a5y17lte": 2,
  "16.0-20190928-nightly-a5y17lte": 2,
  "16.0-20190929-nightly-a5y17lte": 1,
  "16.0-20190930-nightly-a5y17lte": 2,
  "16.0-20191002-nightly-a5y17lte": 6,
  "16.0-20191004-nightly-a5y17lte": 1,
  "16.0-20191005-nightly-a5y17lte": 1,
  "16.0-20191006-nightly-a5y17lte": 3,
  "16.0-20191007-nightly-a5y17lte": 9,
  "16.0-20191008-nightly-a5y17lte": 4,
  "16.0-20191009-nightly-a5y17lte": 2,
  "16.0-20191010-nightly-a5y17lte": 2,
  "16.0-20191012-nightly-a5y17lte": 2,
  "16.0-20191013-nightly-a5y17lte": 7,
  "16.0-20191014-microg-a5y17lte": 1,
  "16.0-20191015-nightly-a5y17lte": 7,
  "16.0-20191017-nightly-a5y17lte": 6,
  "16.0-20191018-nightly-a5y17lte": 4,
  "16.0-20191019-nightly-a5y17lte": 7,
  "16.0-20191020-nightly-a5y17lte": 6,
  "16.0-20191020-unofficial-a5y17lte": 1,
  "16.0-20191021-microg-a5y17lte": 1,
  "16.0-20191022-nightly-a5y17lte": 8,
  "16.0-20191023-nightly-a5y17lte": 2,
  "16.0-20191024-nightly-a5y17lte": 2,
  "16.0-20191025-nightly-a5y17lte": 1,
  "16.0-20191026-nightly-a5y17lte": 10,
  "16.0-20191027-nightly-a5y17lte": 7,
  "16.0-20191028-nightly-a5y17lte": 2,
  "16.0-20191029-nightly-a5y17lte": 1,
  "16.0-20191030-nightly-a5y17lte": 5,
  "16.0-20191101-nightly-a5y17lte": 6,
  "16.0-20191102-nightly-a5y17lte": 14,
  "16.0-20191103-nightly-a5y17lte": 3,
  "16.0-20191104-nightly-a5y17lte": 5,
  "16.0-20191105-nightly-a5y17lte": 4,
  "16.0-20191106-nightly-a5y17lte": 2,
  "16.0-20191107-nightly-a5y17lte": 7,
  "16.0-20191108-nightly-a5y17lte": 5,
  "16.0-20191109-nightly-a5y17lte": 4,
  "16.0-20191110-nightly-a5y17lte": 13,
  "16.0-20191111-microg-a5y17lte": 1,
  "16.0-20191112-nightly-a5y17lte": 13,
  "16.0-20191114-nightly-a5y17lte": 9,
  "16.0-20191115-nightly-a5y17lte": 6,
  "16.0-20191116-nightly-a5y17lte": 4,
  "16.0-20191117-nightly-a5y17lte": 4,
  "16.0-20191118-microg-a5y17lte": 3,
  "16.0-20191118-nightly-a5y17lte": 4,
  "16.0-20191119-nightly-a5y17lte": 10,
  "16.0-20191120-nightly-a5y17lte": 9,
  "16.0-20191121-nightly-a5y17lte": 5,
  "16.0-20191121-unofficial-a5y17lte": 1,
  "16.0-20191122-nightly-a5y17lte": 26,
  "16.0-20191124-nightly-a5y17lte": 15,
  "16.0-20191125-microg-a5y17lte": 8,
  "16.0-20191125-nightly-a5y17lte": 12,
  "16.0-20191126-nightly-a5y17lte": 38,
  "16.0-20191127-nightly-a5y17lte": 37,
  "16.0-20191128-nightly-a5y17lte": 61,
  "16.0-20191129-nightly-a5y17lte": 39,
  "16.0-20191130-nightly-a5y17lte": 42,
  "16.0-20191201-nightly-a5y17lte": 24,
  "16.0-20191202-microg-a5y17lte": 6,
  "16.0-20191202-nightly-a5y17lte": 27,
  "16.0-20191203-nightly-a5y17lte": 30,
  "16.0-20191204-nightly-a5y17lte": 29,
  "16.0-20191205-nightly-a5y17lte": 33,
  "16.0-20191206-nightly-a5y17lte": 34,
  "16.0-20191207-nightly-a5y17lte": 39,
  "16.0-20191208-nightly-a5y17lte": 33,
  "16.0-20191209-microg-a5y17lte": 13,
  "16.0-20191209-nightly-a5y17lte": 29,
  "16.0-20191210-nightly-a5y17lte": 32,
  "16.0-20191212-nightly-a5y17lte": 57,
  "16.0-20191213-nightly-a5y17lte": 9,
  "16.0-20191214-nightly-a5y17lte": 24,
  "16.0-20191215-nightly-a5y17lte": 44,
  "16.0-20191216-microg-a5y17lte": 6,
  "16.0-20191216-nightly-a5y17lte": 27,
  "16.0-20191217-nightly-a5y17lte": 30,
  "16.0-20191218-nightly-a5y17lte": 32,
  "16.0-20191218-unofficial-a5y17lte": 1,
  "16.0-20191219-nightly-a5y17lte": 29,
  "16.0-20191220-nightly-a5y17lte": 39,
  "16.0-20191221-nightly-a5y17lte": 31,
  "16.0-20191222-nightly-a5y17lte": 28,
  "16.0-20191223-microg-a5y17lte": 13,
  "16.0-20191223-nightly-a5y17lte": 26,
  "16.0-20191224-nightly-a5y17lte": 27,
  "16.0-20191225-nightly-a5y17lte": 65,
  "16.0-20191227-nightly-a5y17lte": 25,
  "16.0-20191228-nightly-a5y17lte": 39,
  "16.0-20191229-nightly-a5y17lte": 30,
  "16.0-20191230-microg-a5y17lte": 9,
  "16.0-20191230-nightly-a5y17lte": 35,
  "16.0-20191231-nightly-a5y17lte": 34,
  "16.0-20200101-nightly-a5y17lte": 40,
  "16.0-20200102-nightly-a5y17lte": 29,
  "16.0-20200103-nightly-a5y17lte": 37,
  "16.0-20200104-nightly-a5y17lte": 33,
  "16.0-20200105-nightly-a5y17lte": 26,
  "16.0-20200105-unofficial-a5y17lte": 1,
  "16.0-20200106-microg-a5y17lte": 9,
  "16.0-20200106-nightly-a5y17lte": 17,
  "16.0-20200107-nightly-a5y17lte": 29,
  "16.0-20200108-nightly-a5y17lte": 32,
  "16.0-20200109-nightly-a5y17lte": 23,
  "16.0-20200110-nightly-a5y17lte": 29,
  "16.0-20200111-nightly-a5y17lte": 56,
  "16.0-20200112-nightly-a5y17lte": 22,
  "16.0-20200113-microg-a5y17lte": 10,
  "16.0-20200113-nightly-a5y17lte": 30,
  "16.0-20200114-nightly-a5y17lte": 31,
  "16.0-20200115-nightly-a5y17lte": 44,
  "16.0-20200116-nightly-a5y17lte": 9,
  "16.0-20200117-nightly-a5y17lte": 29,
  "16.0-20200118-nightly-a5y17lte": 56,
  "16.0-20200119-nightly-a5y17lte": 15,
  "16.0-20200120-microg-a5y17lte": 11,
  "16.0-20200120-nightly-a5y17lte": 28,
  "16.0-20200121-nightly-a5y17lte": 28,
  "16.0-20200122-nightly-a5y17lte": 29,
  "16.0-20200123-nightly-a5y17lte": 68,
  "16.0-20200125-nightly-a5y17lte": 96,
  "16.0-20200125-unofficial-a5y17lte": 1,
  "16.0-20200127-microg-a5y17lte": 10,
  "16.0-20200128-nightly-a5y17lte": 30,
  "16.0-20200129-nightly-a5y17lte": 28,
  "16.0-20200130-nightly-a5y17lte": 44,
  "16.0-20200131-nightly-a5y17lte": 32,
  "16.0-20200201-nightly-a5y17lte": 37,
  "16.0-20200202-nightly-a5y17lte": 35,
  "16.0-20200203-microg-a5y17lte": 8,
  "16.0-20200203-nightly-a5y17lte": 66,
  "16.0-20200204-nightly-a5y17lte": 20,
  "16.0-20200205-nightly-a5y17lte": 19,
  "16.0-20200206-nightly-a5y17lte": 31,
  "16.0-20200207-nightly-a5y17lte": 20,
  "16.0-20200208-nightly-a5y17lte": 53,
  "16.0-20200209-nightly-a5y17lte": 38,
  "16.0-20200210-microg-a5y17lte": 7,
  "16.0-20200210-nightly-a5y17lte": 34,
  "16.0-20200211-nightly-a5y17lte": 52,
  "16.0-20200213-nightly-a5y17lte": 42,
  "16.0-20200214-nightly-a5y17lte": 40,
  "16.0-20200215-nightly-a5y17lte": 26,
  "16.0-20200216-nightly-a5y17lte": 36,
  "16.0-20200217-microg-a5y17lte": 13,
  "16.0-20200217-nightly-a5y17lte": 34,
  "16.0-20200218-nightly-a5y17lte": 14,
  "16.0-20200219-nightly-a5y17lte": 16,
  "16.0-20200220-nightly-a5y17lte": 19,
  "16.0-20200221-nightly-a5y17lte": 19,
  "16.0-20200222-nightly-a5y17lte": 49,
  "16.0-20200223-nightly-a5y17lte": 44,
  "16.0-20200224-microg-a5y17lte": 3,
  "16.0-20200225-nightly-a5y17lte": 19,
  "17.1-20200126-unofficial-a5y17lte": 1,
  "17.1-20200209-unofficial-a5y17lte": 8,
  "17.1-20200210-unofficial-a5y17lte": 119,
  "17.1-20200223-unofficial-a5y17lte": 23
}