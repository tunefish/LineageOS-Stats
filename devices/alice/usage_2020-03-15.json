{
  "13.0-20170417-openkirin-alice": 31,
  "13.0-20170418-openkirin-alice": 1,
  "13.0-20170501-openkirin-alice": 12,
  "13.0-20170802-unofficial-alice": 77,
  "14.1": 1,
  "14.1-20170417-openkirin-alice": 3,
  "14.1-20170508-unofficial-alice": 5,
  "14.1-20171015-openkirin-alice": 8,
  "14.1-20171130_210808-unofficial-alice": 6,
  "14.1-20171231_141649-unofficial-alice": 182,
  "14.1-20180624-unofficial-alice": 2,
  "14.1-20180802-unofficial-alice": 2,
  "14.1-20180911-unofficial-alice": 10,
  "14.1-20181001-unofficial-alice": 34,
  "14.1-20181003-unofficial-alice": 3,
  "14.1-20181209-unofficial-alice": 3,
  "14.1-20181215-darkjoker360-alice": 25,
  "14.1-20190203-unofficial-alice": 3,
  "14.1-20190302-darkjoker360-alice": 11,
  "14.1-20190328-darkjoker360-alice": 2,
  "14.1-20190412-darkjoker360-alice": 10,
  "14.1-20190426-darkjoker360-alice": 16,
  "14.1-20190519-darkjoker360-alice": 15,
  "14.1-20190616-darkjoker360-alice": 38,
  "14.1-20190806-unofficial-alice": 12,
  "14.1-20190906-unofficial-alice": 66,
  "14.1-20191119-unofficial-alice": 145,
  "14.1-20191202-unofficial-alice": 1,
  "14.1-20191207-unofficial-alice": 1,
  "14.1-20191213-unofficial-alice": 4,
  "14.1-20191224-unofficial-alice": 288,
  "14.1-20200102-darkdroiddev-alice": 1,
  "14.1-20200215-unofficial-alice": 75,
  "15.1-20190509-darkjoker360-alice": 12,
  "15.1-20191201-darkjoker360-alice": 160,
  "15.1-20191202-unofficial-alice": 1,
  "15.1-20200217-darkjoker360-alice": 11,
  "8.1-20190925-pre-stable-alice": 1
}