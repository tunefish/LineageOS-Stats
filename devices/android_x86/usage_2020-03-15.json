{
  "14.1-20170228-unofficial-android_x86": 29,
  "14.1-20191127-agree-android_x86": 2,
  "14.1-20191128-agree-android_x86": 5,
  "14.1-20191129-agree-android_x86": 2,
  "14.1-20200120-unofficial-android_x86": 4,
  "14.1-20200122-unofficial-android_x86": 7,
  "14.1-20200124-unofficial-android_x86": 1,
  "14.1-20200127-unofficial-android_x86": 1,
  "14.1-20200128-unofficial-android_x86": 10,
  "14.1-20200214-unofficial-android_x86": 3,
  "14.1-20200215-unofficial-android_x86": 1,
  "14.1-20200216-unofficial-android_x86": 1,
  "14.1-20200218-nicov-android_x86": 1,
  "14.1-20200224-nicov-android_x86": 2,
  "14.1-20200224-zts-android_x86": 1,
  "14.1.0-android_x86": 3833
}