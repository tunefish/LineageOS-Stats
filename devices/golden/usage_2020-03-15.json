{
  "12.1-20170404-unofficial-golden": 1,
  "13.0-20170118-unofficial-golden": 2,
  "13.0-20170121-unofficial-golden": 6,
  "13.0-20170129-unofficial-golden": 98,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 15,
  "13.0-20170310-unofficial-golden": 42,
  "13.0-20170328-unofficial-golden": 704,
  "13.0-20170420-unofficial-golden": 30,
  "13.0-20180315-unofficial-golden": 1,
  "13.0-20190315-unofficial-golden": 1,
  "14.1-20170122-unofficial-golden": 206,
  "14.1-20170318-unofficial-golden": 4,
  "14.1-20170402-unofficial-golden": 3,
  "14.1-20170415-unofficial-golden": 2,
  "14.1-20170419-unofficial-golden": 5,
  "14.1-20170503-unofficial-golden": 2,
  "14.1-20170507-unofficial-golden": 21,
  "14.1-20170516-unofficial-golden": 1,
  "14.1-20170522-unofficial-golden": 5,
  "14.1-20170609-unofficial-golden": 575,
  "14.1-20170610-unofficial-golden": 10,
  "14.1-20170628-unofficial-golden": 823,
  "14.1-20170724-unofficial-golden": 35,
  "14.1-20170817-unofficial-golden": 71,
  "14.1-20170920-unofficial-golden": 12,
  "14.1-20171011-unofficial-golden": 89,
  "14.1-20171215_192311-official-golden": 71,
  "14.1-20171215_192311-unofficial-golden": 1065,
  "14.1-20180109_002536-unofficial-golden": 311,
  "14.1-20181014-unofficial-golden": 3,
  "14.1-20200222-unofficial-golden": 1,
  "15.1-20181030-unofficial-golden": 7
}